scripts/build_android.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash -e
+
+ROOT_PATH=`dirname "${0}"`/..
+
+# Android
+echo "Building Android"
+
+cd $ROOT_PATH/android
+npx appc run -p android --build-only
+
+echo "Finished building Android"
+
+cd -
```

Comment:
I like to be explicit in these, in case some other `cd` creeps into the script.
```suggestion
cd $ROOT_PATH
```

Reply:
Root path is not the original destination so this is wrong

Reply:
Oh, yeah. Oops. I need to have my eyes checked. Note, this only matters if you have a script call this script, otherwise the script runs in its own environment and none of the `cd`s apply to the calling environment.